### PR TITLE
Add blastp PresetProgram

### PR DESCRIPTION
**Pull request by jvfe**
Changes made:
- Add PresetProgram for blastp + tests
- Apply black styling to non-formatted modules

Just something I totally forgot to do in the first blast PR.

### CODE REVIEW

File: bioprov/tests/test_bioprov_programs.py [hunk 1]
```diff
@@ -55,6 +63,34 @@ def test_blastn():
...
+    blast = blastn(s, reference_db)
+    blast_params = blast.serializer()["params"]
+
+    expected = ["-db", "-outfmt", "-query", "-out"]
+
+    assert list(blast_params.keys()) == expected
+    assert blast_params["-outfmt"]["value"] == "6"
+
+
+def test_blastp():
```
Comment (jvfe) [inline bioprov/tests/test_bioprov_programs.py:80]: I really don't like how there is so much repetition in the program tests but I don't see any other way to do it 😔. Any ideas?

Comment (vinisalazar) [reply bioprov/tests/test_bioprov_programs.py:80]: Tests need major refactoring. The ideal would be to have class-based testing, and create fixtures for a 'full' and a 'short' test suite. However this is not a priority at the moment. The package has good coverage, and improvements like this should come in a future major version, e.g. BioProv v1.x.x

**Review by vinisalazar — APPROVED** [review]
LGTM, thanks @jvfe ! I will rebase the dev branch prior to merging.

File: bioprov/tests/test_bioprov_programs.py (hunk 1 above)
Comment (jvfe) [reply bioprov/tests/test_bioprov_programs.py:80]: Oh for sure! It would mean a lot of changes. What do you mean by 'full' and 'short' suites?

**Pull request merged**